components/mpas-ocean/cime_config/buildnml

```diff
@@ -807,6 +809,8 @@ def buildnml(case, caseroot, compname):
             lines.append('    <var name="columnIntegratedSpeed"/>')
             lines.append('    <var name="barotropicSpeed"/>')
             lines.append('    <var name="landIceFreshwaterFlux"/>')
+            lines.append('    <var name="landIceFreshwaterFluxTotal"/>')
+            lines.append('    <var name="frazilIceFreshwaterFlux"/>')
```

Comment:
I think this one analysis member (`highFrequencyOutput`) probably should only have `landIceFreshwaterFluxTotal` and not the constituents.  We want to keep it pretty light weight.

Reply:
I agree